2216/2/rapper.rb

```diff
@@ -0,0 +1,26 @@
+class Rapper
+  attr_reader :rapper
+
+  def initialize(participant_name)
+    @rapper = participant_name
+  end
+
+  def battles
+    fetch_battles
+  end
+
+  def self.right_battle?(file_name, rapper)
```

Comment:
What does `right_battle?` mean and why this is a class method?

Given we have a rapper object.
`rapper = Rapper.new(..)`
And we want to check that he is the _right_ (still don't understand what it means) for a battle
`rapper.right?(battle)`

That's it. You don't need to use class methods there. Think about your class as a business entity.